code/game/gamemodes/cult/runes.dm

```diff
@@ -535,7 +535,7 @@ var/list/teleport_runes = list()
 	if(rune_in_use)
 		return
 	for(var/mob/living/M in orange(1,T))
-		if(M.stat == DEAD)
+		if(M.stat == DEAD && !iscultist(M))
```

Comment:
Maybe it should only be humans with a ckey attached then (and it could check against ghosts.current like soulstones do), or else you could just use mining mobs, monkey cubes, slimes, etc